microsoft-edge/devtools-guide-chromium/progressive-web-apps/simulate-window-controls-overlay.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Simulate a Window Controls Overlay in a PWA's title bar area
+description: Display content in a PWA's title bar area by using the Window Controls Overlay API.
```

Comment:
Proposal. This is really what the value proposition is.

```suggestion
description: Simulate enabling the Window Controls Overlay API in your Progressive Web App (PWA), to test your styles, without installing the PWA first.
```

Reply:
```
description: Simulate enabling the Window Controls Overlay API in 
your Progressive Web App (PWA), to test your CSS styles, 
without having to install the PWA first.
```